src/Components/benchmarkapps/BlazingPizza.Server/ssr-benchmarks.yml

```diff
@@ -0,0 +1,81 @@
+imports:
+    - https://raw.githubusercontent.com/dotnet/crank/main/src/Microsoft.Crank.Jobs.Bombardier/bombardier.yml
+
+jobs:
+  server:
+    source:
```

Comment:
Overall I am impressed you got it working this way. It's not the most straightforward technique. But also it's not the ideal one. The recommendation is to use the github based source, and keep `localFolder` when you don't want to push your local changes, like for some temporary work. Also `localFolder` is usually used to send source code, though it definitely works with the whole built folder too like you did. An optimal solution could be to have two jobs, one called `blazorpizza` and one called `blazorpizza_local`. Both pointing to the source (GH and local).

What it will allow is for crank to use ANY sdk/coreclr/aspnet, even nightlies. And you can still use the `[job].options.outputFiles c:\build\myd.dll` if you want to override the sdk/runtimes files without your locally built ones (can be one or more files).

Reply:
I've tried various ways to use `outputFiles` but haven't managed to make it work. For example:

```
jobs:
  server:
    options:
      outputFiles:
        - ../../../../artifacts/bin/BlazingPizza.Server/Release/net8.0/**;
```

... does upload the right locally-build files, but fails with a nullref, presumably because I didn't upload any source (because I don't need it).

Adding:

```
    source:
      localFolder: ./
```

... seems to result in it hanging for a long time before reporting `Requests/sec           | 0` (I think it can't start the app for some reason). Changing it to:

```
    source:
      repository: https://github.com/aspnet/aspnetcore
```

... results in this error:

```
[04:39:58.026] 'application' is now building ... http://asp-perf-lin:5001/jobs/10/buildlog
[04:40:05.404] 'application' failed on agent, stopping...
An unexpected error occurred while building the job: One or more errors occurred. (An error occurred trying to start process '/tmp/benchmarks-agent/benchmarks-server-1/3lq3rmlt.gs5/dotnet' with working directory '/tmp/benchmarks-agent/benchmarks-server-1/mn2ub1zx.moq/aspnetcore/published'. No such file or directory)
```

So altogether it feels like the original technique with `localFolder: ../../../../artifacts/bin/BlazingPizza.Server/Release/net8.0` was much more effective.

Do you think it's important to pursue an alternative here? You mentioned that *localFolder is usually used to send source code, though it definitely works with the whole built folder too like you did* so given that this works and is expected to work, perhaps it would be reasonable to leave it as-is!

Reply:
Resolving as the benchmarks aspect is now handled in https://github.com/aspnet/Benchmarks/pull/1804